src/pepr/operator/controllers/istio/injection.ts

```diff
@@ -1,7 +1,11 @@
-import { K8s, Log, kind } from "pepr";
+import { K8s, kind } from "pepr";
 
+import { childLog } from "../../../logger";
 import { UDSPackage } from "../../crd";
 
+// configure subproject logger
+const log = childLog("operator.controllers.istio");
+
```

Comment:
I'm leaning towards shorter, slightly more generic titles here...maybe like `operator.istio`? Do you think that would be more confusing? Part of my thinking is that the end user viewing these logs probably doesn't need to know the nitty-gritty of the directory/file structure, but it's probably helpful to capture:
- Is this policy code, operator code (`Package` CR related), or "other"?
- If operator related - what piece of the setup is this (at minimum istio vs sso vs monitoring vs networkpolicies?)
- If other - what is it? (probably come up with some standard of how we name these)
- If policy related - maybe just policy is sufficient or we differentiate between exemption related pieces and validation pieces?

Reply:
I'm thinking the naming scheme would be the top level directory name ( currently these: `istio`, `operator`, `policies`, `prometheus` ) followed by the directory name where the file is found seprated by '.' ( ex. `operator.istio` or `policies.exemptions` ). Essentially what you have and addresses the "other" usecases.

Reply:
I like that yeah. Should be enough information without being too fine-grained, and we can always adjust these or add additional fields as needed.